scripts/testone.sh

```diff
@@ -0,0 +1,27 @@
+set -ex
+
+# File assumes tat you are running the program at the program at the root directory of the coda repo
+
+if [[ "$DUNE_PROFILE" -eq "" ]]; then
+    DUNE_PROFILE=dev
+fi
+
+ABSOLUTE_FILE_PATH="$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"
```

Comment:
Can you add a check to make sure we pass a parameter to the script (something like if $1 is not blank, print usage, exit 1)